utils/doclint/generateDotnetApi.js

```diff
@@ -458,18 +458,16 @@ function renderMethod(member, parent, output, name) {
    * @param {Documentation.Member} argument 
    */
   const pushArg = (innerArgType, innerArgName, argument) => {
-    let isEnum = enumTypes.has(innerArgType);
     let isNullable = nullableTypes.includes(innerArgType);
+
     const requiredPrefix = argument.required ? "" : isNullable ? "?" : "";
     const requiredSuffix = argument.required ? "" : " = default";
     args.push(`${innerArgType}${requiredPrefix} ${innerArgName}${requiredSuffix}`);
   };
 
   let parseArg = (/** @type {Documentation.Member} */ arg) => {
     if (arg.name === "options") {
-      arg.type.properties.forEach(prop => {
-        parseArg(prop);
-      });
+      arg.type.properties.forEach(parseArg);
```

Comment:
should this not be `sortedProperties()`? I might be missing something though :)

Reply:
All the changes I'm making in this PR is so we have the properties in the original order in dotnet (use `properties` instead of `sortedProperies`)